lib/rubygems/specification_policy.rb

```diff
@@ -219,6 +221,15 @@ def validate_dependencies # :nodoc:
     end
   end
 
+  def validate_required_ruby_version
+    op, dep_version = @specification.required_ruby_version.requirements.first
+    segments = dep_version.segments
+
+    if (op == '>' || op == '>=') && segments == [0]
+      warning "please specify minimal required ruby for your RubyGem"
```

Comment:
I would go with something like

```suggestion
      warning "make sure you specify the oldest ruby version that you want your gem to support by setting the `required_ruby_version` gemspec attribute
```